.github/workflows/e2e.yml

```diff
@@ -7,7 +7,11 @@ on:
 jobs:
   playwright:
     name: 'Playwright Tests'
+    timeout-minutes: 60
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
```

Comment:
Why test multiple node versions? Let's stick to LTS and save GHA minutes, it's not a node serverside app...